Islandora-CLAW/islandora_demo
# Features export for https://github.com/Islandora-CLAW/CLAW/issues/905

**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/905

* Goes with https://github.com/Islandora-Devops/migrate_7x_claw/pull/10

# What does this Pull Request do?

Features export for working with original files, since in https://github.com/Islandora-CLAW/CLAW/issues/905 we're tagging OBJs as Original Files instead of Preservation Masters

# How should this be tested?

See testing instructions at https://github.com/Islandora-Devops/migrate_7x_claw/pull/10

# Interested parties
@Islandora-CLAW/committers
